easybuild/easyconfigs/f/FIAT/FIAT-2019.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'FIAT'
+version = '2019.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bitbucket.org/fenics-project/fiat'
+description = """The FInite element Automatic Tabulator (FIAT) supports
+generation of arbitrary order instances of the Lagrange elements on
+lines, triangles, and tetrahedra. It is also capable of generating
+arbitrary order instances of Jacobi-type quadrature rules on the same
+element shapes."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
```

Comment:
@akesandgren FIAT depends on SymPy?
see https://fenics.readthedocs.io/projects/fiat/en/latest/installation.html?highlight=sympy#requirements-and-dependencies

Reply:
Ohh, did I miss that... I went through the install instructions multiple times...

Reply:
No I didn't. See the exts_list it has sympy 1.6

Reply:
Although I currently can't remember why i put it in ext_lists instead of a standard dep...
I'll look at that...

Reply:
Done, now using external sympy 1.5.1